final/app/src/main/java/org/tensorflow/lite/examples/digitclassifier/DigitClassifier.kt

```diff
@@ -56,19 +57,21 @@ class DigitClassifier(private val context: Context) {
     var interpreter: Interpreter
     if (model is ByteBuffer) {
       interpreter = Interpreter(model, options)
-    } else {
-      interpreter = Interpreter(model as File, options)
+    } else if (model is CustomModel) {
```

Comment:
Is it possible for the `model` to be of a different type than one of these two? If so, we won't load the `interpreter`, is that a problem?

Reply:
Handled all cases in latest commit.